common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1553,6 +1553,11 @@
     },
     "sqlState" : "HY109"
   },
+  "INVALID_DATABASE_NAME" : {
```

Comment:
Can't you re-use the existing error class: `INVALID_SCHEMA_OR_RELATION_NAME`? If not, could you make it more specific.

Reply:
Maybe we can re-use error class: `REQUIRES_SINGLE_PART_NAMESPACE`?
But the prompt message is called `namespace` instead of `database`, seems to be okay? I'm too sure
https://github.com/apache/spark/blob/376de8a502fca6b46d7f21560a60024d643144ea/common/utils/src/main/resources/error/error-classes.json#L2784-L2789

Reply:
SGTM